src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -2428,4 +2435,42 @@ static OrganizationUserStatusType GetPriorActiveOrganizationUserStatusType(Organ
 
         return status;
     }
+
+    public async Task CreatePendingOrganization(Organization organization, string ownerEmail)
+    {
+        organization.Enabled = false;
+        organization.Status = OrganizationStatusType.Pending;
+        await _organizationRepository.CreateAsync(organization);
+
+        var ownerOrganizationUser = new OrganizationUser
+        {
+            OrganizationId = organization.Id,
+            UserId = null,
+            Email = ownerEmail,
+            Key = null,
+            Type = OrganizationUserType.Owner,
+            Status = OrganizationUserStatusType.Invited,
+            AccessAll = true,
+            CreationDate = organization.CreationDate,
+            RevisionDate = organization.CreationDate
+        };
+        await _organizationUserRepository.CreateAsync(ownerOrganizationUser);
+
+        await _eventService.LogOrganizationUserEventAsync(ownerOrganizationUser, EventType.OrganizationUser_Invited);
```

Comment:
Always log the event after the event has happened - this should be after `SendInviteAsync`.